template/{{ project_slug }}.opam

```diff
@@ -13,7 +13,6 @@ bug-reports: "https://github.com/{{ github_username }}/{{ project_slug }}/issues
 depends: [
   "ocaml" {>= "4.08.0"}
   "dune"
-  "jsoo-react"
```

Comment:
We still need this I think. It doesn't matter too much because the file is re-generated by dune, but I think the generated version will have a dep on `jsoo-react`